README.md

```diff
@@ -39,54 +39,38 @@ ant test
 
 See the [documentation for api usage].
 
-After a user completes the oauth2 flow, the credentials for their account will be stored as a data source with a configured folder created in the user's root mailbox during authentication. If an import class is associated, it can be triggered manually with a `zmsoap` import data request.
+After a user completes the oauth2 flow, the credentials for their account will be stored as a data source with a configured folder created in the user's root mailbox during authentication. An import can be triggered manually with a `zmsoap` import data request.
 
 ---
 
 ## Configuration
 
-This service's configuration can all be found in Zimbra's `localconfig.xml` file (usually located at `/opt/zimbra/conf/localconfig.xml`)
-
-**Localconfig General Properties**
+This service's configuration for OAuth clients are setup in Ldap attributes on a global or domain level.
 
+**Ldap Properties**
 
-| Key | Description | Optional | Example Options |
-| --- | ----------- | -------- | --------------- |
-| host_uri_template | The host uri to connect via ZMailbox | Yes | `https://%s:443` |
-| zm_oauth_classes_handlers_yahoo<sup>1</sup> | The handler implementation class for the client | | `com.zimbra.oauth.handlers.impl.YahooOAuth2Handler` |
+See the [client setup wiki].
 
-<sup>1</sup>Replace the `yahoo` part of the key name with the name of the client (e.g. `yahoo`, `google`, `outlook`).
+| Key | Description | Required By | Template |
+| --- | ----------- | ----------- | -------- |
+| zimbraOAuthConsumerCredentials | OAuth credentials for a client, set at global config or domain level. | All | `<client-id>:<client-secret>:<client>` |
+| zimbraOAuthConsumerRedirectUri | The callback where the client returns the user too. | All | `http[s]://<domain[:port]>/service/extension/oauth2/authenticate/<client>:<client>` |
+| zimbraOAuthConsumerAPIScope | The scopes required to access user data. | Google, Facebook | `<scope1>,<scope2>,...:<client>` |
```

Comment:
Don't use comma separation in the example only one of the clients uses that - use `+` since that's the primary.